pkg/volume/csi/csi_plugin.go

```diff
@@ -912,9 +923,13 @@ func (p *csiPlugin) getFSGroupPolicy(driver string) (storage.FSGroupPolicy, erro
 	if p.csiDriverLister != nil {
 		kletHost, ok := p.host.(volume.KubeletVolumeHost)
 		if ok {
-			if err := kletHost.WaitForCacheSync(); err != nil {
+			synced, err := kletHost.CSIDriversSynced()
+			if err != nil {
 				return storage.ReadWriteOnceWithFSTypeFSGroupPolicy, err
 			}
+			if !synced {
+				return storage.ReadWriteOnceWithFSTypeFSGroupPolicy, errors.New("CSIDrivers has not bean synced yet")
```

Comment:
"bean"->"been"

Reply:
Thanks for your suggestion, I have modified the code.